.travis.yml

```diff
@@ -77,7 +77,7 @@ script:
   - export CXX="${CXX}-${COMPILER_VERSION}"
   - cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DRC_ENABLE_TESTS=ON -DRC_ENABLE_EXAMPLES=ON -DRC_ENABLE_GTEST=ON -DRC_ENABLE_GMOCK=ON -DRC_ENABLE_BOOST=ON -DRC_ENABLE_BOOST_TEST=ON -G Ninja ..
   - ninja -j1 # Don't run out of memory, please
-  - ctest --output-on-failure
+  - travis_wait ctest --output-on-failure
```

Comment:
Yeah, they're pretty slow to be honest. Did you have any particular optimizations in mind?

I've been thinking about simply optimizing the command sequence generator itself. Right now, a lot of time is spent rebuilding the sequence after small changes... if we could apply some immutable data structure with sharing, maybe it would be faster. It's a very functional problem after all.